docs/community/bindings.md

```diff
@@ -0,0 +1,7 @@
+# Bindings
+
+As a C library, bindings can be made to call H3 functions from different programming languages. This page lists different bindings currently available.
+
+## R
+
+- [scottmmjackson/h3r](https://github.com/scottmmjackson/h3r)
```

Comment:
I know I pointed it out, but this binding has little of the API exposed (yet) and I'm not sure if the author wants to actually support it long-term.

We should ask them if they want a shout-out from this repository (and release the in house bindings, please! :) )

Reply:
@scottmmjackson Could you let us know your thoughts on adding the link here?

I don't think bindings need to have a lot of the API exposed to be mentioned here. I can also understand starting the Community section as blank and leaving a note asking maintainers to make a pull request to add their link to this page.

(I'm working on it. :))

Reply:
I'm for it. I think it'll definitely give me a push to maintain it better.

I'd also be open to pointers about any additional methods y'all think should be exposed, or whether the method names should be changed to be more in line with the C API. 

My immediate thoughts were that folks would want to index coordinates and figure out what the bounding hex of an index was. I hadn't thought through many use cases outside of that, though I can imagine they exist.

Reply:
It's also worth mentioning that @pocket7878 and I have duplicated efforts for Rust bindings:

https://github.com/pocket7878/h3-rs

https://github.com/scottmmjackson/h3api

Reply:
Thanks @scottmmjackson, and thanks for starting the R bindings! I'll keep the h3r link as the first one and add a note that PRs are welcome.

I think it would be really helpful to add a document to the H3 developer documentation that talks about making bindings (and has examples :) ) That's something I can do in the next couple days. We have a few things in the repo specifically for binding authors, like the `make binding-functions` target for making a list of functions in the public API. That binding functions list is the 'complete' list of functions to bind, but obviously different languages have different expectations about how those functions are named (capitalized, usually), what types of arguments, and so on.

Reply:
That would be very helpful. For Rust, I built this list by cribbing from the doxygen docs. 

https://github.com/scottmmjackson/h3api/blob/master/build.rs#L20

I'd expected it might be different for R just because I want to limit how many times I cross the FFI so I can offer users more speed while sort of compromising more fine-grained utility by anticipating their workflows a little bit.

pocket7878 just sucked the whole thing directly through bindgen. My approach was to make the `cargo doc` easier to read.